reflex/utils/exceptions.py

```diff
@@ -19,3 +19,13 @@ class MatchTypeError(TypeError):
     """Raised when the return types of match cases are different."""
 
     pass
+
+class ExceptionHandler:
```

Comment:
What benefits do you expect from this separate class?

Reply:
I thought of this class as an exception handler that could be used not only for event handling but also globally in any other place where custom exception handling is needed. Later I did not implement it.